wiki/Article_styling_criteria/Formatting/ko.md

```diff
@@ -0,0 +1,903 @@
+| :-- | :-- | :-- |
+| `en.md` | 영어 | English |
+| `ar.md` | 아랍어 | اَلْعَرَبِيَّةُ |
+| `be.md` | 벨라루스어 | Беларуская мова |
+| `bg.md` | 불가리아어 | Български |
+| `cs.md` | 체코어 | Česky |
+| `da.md` | 덴마크어 | Dansk |
+| `de.md` | 독일어 | Deutsch |
+| `el.md` | 그리스어 | Ελληνικά |
+| `es.md` | 스페인어 | Español |
+| `fi.md` | 핀란드어 | Suomi |
+| `fr.md` | 프랑스어 | Français |
+| `hu.md` | 헝가리어 | Magyar |
+| `id.md` | 인도네시아어 | Bahasa Indonesia |
+| `it.md` | 이탈리아어 | Italiano |
+| `ja.md` | 일본어 | 日本語 |
+| `ko.md` | 한국어 | 한국어 |
+| `nl.md` | 네덜란드어 | Nederlands |
+| `no.md` | 노르웨이어 | Norsk |
+| `pl.md` | 폴란드어 | Polski |
+| `pt.md` | 포르투갈어 | Português |
+| `pt-br.md` | 포르투갈어 (브라질) | Português (Brasil) |
+| `ro.md` | 루마니아어 | Română |
+| `ru.md` | 러시아어 | Русский |
+| `sk.md` | 슬로바키아어 | Slovenčina |
+| `sv.md` | 스웨덴어 | Svenska |
+| `th.md` | 태국어 | ไทย |
+| `tr.md` | 터키어 | Türkçe |
+| `uk.md` | 우크라이나어 | Українська мова |
+| `vi.md` | 베트남어 | Tiếng Việt |
+| `zh.md` | 중국어 (간체) | 简体中文 |
+| `zh-tw.md` | 중국어 (번체, 대만) | 繁體中文（台灣） |
+
+*참고: 웹사이트는 유저가 선택한 언어의 문서를 표시합니다. 표시하지 못할 경우에는 영어 문서로 표시됩니다.*
+
+## 머리말
+
+머리말은 반드시 파일의 맨 위에 위치해야 하며, [YAML](https://en.wikipedia.org/wiki/YAML#Example)로 작성 된, 문서의 추가 정보를 가지고 있습니다. 이것은 반드시 3개의 하이픈 (`---`)으로 감싸져 있어야 하며, 타이틀의 제목이 시작되기 전 한줄의 빈 공간이 있어야 합니다.
+
+### 도움이 필요한 문서
+
+*참고: 이 태그가 있는 영어 문서는 번역하지 마세요. 이 외에도, 번역 자체를 정리해야 할 필요가 있다면 이 태그를 추가하세요.*
+
+`need_cleanup` 태그는 다시 작성해야 하거나 문서 서식에 도움이 필요할 경우 추가할 수 있습니다. 이를 위해 GitHub에서 이슈를 여는 것도 허용됩니다. 태그를 사용할 때는 아래와 같이 작성해야 합니다:
+
+```yaml
+needs_cleanup: true
+```
+
+문서에 이 태그를 추가하는 경우, 태그를 제거하기 위해 해야하는 작업들을 설명하는 [주석](#주석)을 추가해야 합니다.
+
+### 오래된 문서
+
+*참고: 이 태그가 있는 영어 문서는 번역하지 마세요. 만약 영어 문서가 이 태그를 가지고 있다면, 번역된 문서에도 반드시 이 태그를 추가해야 합니다.*
+
+영어 문서에 포함된 내용이 오해의 소지가 있거나, 더 이상 관련이 없는 내용을 포함하는 경우에는 오래된 문서로 처리할 수 있습니다. 이 경우 `outdated` 태그를 추가해야 하며, 다음과 같이 작성을 해야 합니다:
+
+```yaml
+outdated: true
+```
+
+이 태그를 문서에 추가하는 경우, 태그를 제거하기 위해 업데이트되어야 할 내용을 설명하는 [주석](#주석)을 추가해야 합니다.
+
+### 오래된 번역
+
+영어 문서에 변경점이 있을 경우 번역된 문서들에는 `outdated_translation` 태그를 사용해야 합니다. 문서에 사소한 단어나 문법 변경과 같이 문서에 영향을 주지 않는 경우에는 제외합니다.
+
+```yaml
+outdated_translation: true
+```
+
+번역이 오래되었을 경우, 영어 버전이 업데이트된 첫번째 커밋을 표시하는 `outdated_since` 태그를 반드시 추가해야 합니다.
+
+```yaml
+outdated_since: 29eac89cd535f8b071ca000af8fe4f0be22bdc9b
+```
+
+### 문서 태그하기
+
+태그는 웹사이트의 검색 엔진이 문서를 더 잘 가져오도록 도와줍니다. 태그는 작성된 문서와 동일한 언어로 작성되어야 하며, 원본 문서의 태그 목록을 포함해야 합니다. 태그는 해당되는 경우에 소문자를 사용하세요.
+
+예로, "Beatmap discussion"이라는 문서에서는 다음과 같은 태그를 가지고 있습니다:
+
+```yaml
+tags:
+  - beatmap discussions
+  - modding V2
+  - MV2
+```
+
+### 검토가 없는 번역
+
+*참고: 위키 관리자가 병합하기 전 이 마크를 추가할지 결정합니다.*
+
+다른 원어민의 검토 없이 번역이 위키에 추가될 때가 있습니다. 이 경우, 다음에 번역을 할 번역가들이 다시 한번 체크할 필요가 있다는 것을 알리기 위해 `no_native_review` 태그가 추가됩니다. 태그는 다음과 같이 작성해야 합니다:
+
+```yaml
+no_native_review: true
+```
+
+### 문서 이름 짓기
+
+*또한 [폴더 이름](#폴더-이름)와 [제목](#제목)도 참조하세요.*
+
+문서의 제목은 단수여야 하며, 문장의 첫 글자를 대문자로 작성해야 합니다. 자세한 내용은 [Wikipedia's naming conventions article](https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(plurals))을 확인하세요.
```

Comment:
the link here should be translated, the whole thing isn't a name or anything

Reply:
There is no translated wiki page, should find the other ways..

Reply:
i think you misunderstood. "Wikipedia's naming conventions article" isn't an article title.

to illustrate, in swedish i'd translate it to "Wikipedias namnkonvention för artiklar"